cmd/armada/export/logs/exportlogs.go

```diff
@@ -0,0 +1,57 @@
+package logs
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/armada/pkg/defaults"
+	kind "sigs.k8s.io/kind/pkg/cluster"
+)
+
+// ExportLogsFlagpole is a list of cli flags for export logs command
+type ExportLogsFlagpole struct {
+	Clusters []string
+}
+
+// ExportLogsCommand returns a new cobra.Command under export command for armada
+func ExportLogsCommand(provider *kind.Provider) *cobra.Command {
+	flags := &ExportLogsFlagpole{}
+	cmd := &cobra.Command{
+		Args:  cobra.NoArgs,
+		Use:   "logs",
+		Short: "Export kind cluster logs",
+		Long:  "Export kind cluster logs",
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			// remove existing before exporting
+			_ = os.RemoveAll(filepath.Join(defaults.KindLogsDir, defaults.KindLogsDir))
```

Comment:
once again you are not reporting nor handling errors. In this case for sure it needs to handle because as per this line I'm assuming you expect the folder not to exist for the rest of the command,  thus you need to act accordingly.